packages/next/src/lib/metadata/default-metadata.tsx

```diff
@@ -1,10 +1,27 @@
-import type { ResolvedMetadata } from './types/metadata-interface'
+import type {
+  ResolvedMetadata,
+  ResolvedViewport,
+} from './types/metadata-interface'
+
+export function createDefaultViewport(): ResolvedViewport {
+  return {
+    // name=viewport
+    width: 'device-width',
+    initialScale: 1,
```

Comment:
all keys should be present here since the Resolved type should have no optional members

Reply:
Yeah I shared the type there because it's almost the same, another resolved viewport layout type might introduce a few extra bytes but very little gain there